android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -894,8 +894,8 @@
 		"startBias": ["Coast"],
 		"preferredVictoryType": "Cultural",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Magnificent Empress Theodora, beloved of Byzantium and of Rome, your people salutes you! With your beauty and intelligence, you rose up from actress and courtesan to the rank of the most powerful woman in the Roman Empire, as wife of Justinian I. In the late 520's AD, you helped your husband to engage spiritual and legal reforms, promugalting numerous laws to improve women status and treatment in the empire. You also made of Constantinople the jewel of the Eastern Roman Empire with the restoration and construction of many aqueducts, bridges, and churches, its beauty culminating with the creation of the Hagia Sophia, one of the most splendid piece of architecture in the world.",
```

Comment:
Your people salute you
You helped your husband engage in
To improve women's status